### DISCUSSION TITLE
covid vs 電腦模型

### DISCUSSION THREAD

**Discussion by eason8479** [1101-GS4719-C/Python-Community-GS4719-C#850] [General]
https://www.youtube.com/watch?v=H_akzwzghWQ

雖然這影片主要是在說covid對航班的影響，不過裡面有講到一些和電腦相關的

簡單來說，航空公司為了賺取最大利潤，價格都是浮動的，一開始這當然都是人在操作的
隨時間演進，航空公司都利用蒐集來的資料，搭配電腦模型，將其外包給電腦了
不過由於covid的影響，導致過去的資料基本上失效了，而電腦也因此做出了不合理的決策

但是航空公司也不是笨蛋，至少在疫情完全回復前，暫時回歸人為定價
這不禁讓人思考，如果在未來類似的事情再次發生時，我們還有沒有機會回歸人為

**Comment by ching020721** [comment]
特定事件發生導致先前的大數據失效，回歸人為定價很正常，所以未來又有相關情況發生，我認為回歸人為的可能性應該蠻高的

**Reply by eason8479** [reply]
回歸人為當然是面對這種問題的最佳解
但我文末的擔憂是到那時我們還有沒有這個選擇

假設接下來幾代人都平平安安，沒有遇上這些特定事件
而當某天數據失效時
大家才發現，已經沒有人會做這些被我們外包給AI的工作

**Comment by williamyen0623** [comment]
原來在covid之前的定價是由電腦模型決定的，感謝版主的分享，我現在才知道
看來covid還造成了很多我們意想不到的影響

**Comment by CPCpyt** [comment]
我覺得技術是看人和情況使用的，
對於這種黑天鵝事件，運用本來的技術本身就不合理。
